pkg/ui/workspaces/cluster-ui/src/util/format.ts

```diff
@@ -172,10 +172,12 @@ export const DurationFitScale =
 export const DATE_FORMAT = "MMM DD, YYYY [at] H:mm";
 
 /**
- * Alternate 24 hour UTC format
+ * Alternate 24 hour UTC formats
  */
 export const DATE_FORMAT_24_UTC = "MMM DD, YYYY [at] H:mm UTC";
 export const DATE_WITH_SECONDS_FORMAT_24_UTC = "MMM DD, YYYY [at] H:mm:ss UTC";
+export const DATE_WITH_SECONDS_AND_MILLISECONDS_FORMAT_24_UTC =
```

Comment:
keep this format (since you use for summary cards) and create another without the `UTC` part. :)